src/bootstrap/src/core/build_steps/compile.rs

```diff
@@ -40,8 +42,9 @@ pub struct Std {
     ///
     /// This shouldn't be used from other steps; see the comment on [`Rustc`].
     crates: Vec<String>,
-    /// When using download-rustc, we need to use a new build of `std` for running unit tests of Std itself,
-    /// but we need to use the downloaded copy of std for linking to rustdoc. Allow this to be overridden by `builder.ensure` from other steps.
+    /// When using `download-rustc`, we need to use a new build of `std` for running unit tests of
+    /// std itself, but we need to use the downloaded copy of std for linking to rustdoc. Allow this
+    /// to be overridden by `builder.ensure` from other steps.
```

Comment:
Unrelated comment reflow diffs, was annoying when reading this locally, I'll drop these formatting diffs if we want to merge the actual changes.